### ISSUE TITLE
Accordian Box: Add API for passing descriptive paragraph

### ISSUE THREAD

**Issue by kathy-phet** [phetsims/sun#915]
When an accordian box contains a complex node that isn't available in the pdom, we should be able to put a descriptive paragraph there in the pdom - similar to the way alt-text is used for images.

**Comment by jessegreenberg** [comment]
I don't think we should add options to AccordionBox (and other content containers) for this. I think that the "alt-text" content should be implemented on the content Node of the AccordionBox.

With the low level API, that looks something like
```js
energyLevelDiagram.tagName = 'p';
energyLevelDiagram.innerContent = 'This is a description of the energy level diagram...';
```

More broadly, we should think about the high level API for this for any Node. We could keep using `accessibleName`

```js
energyLevelDiagram.accessibleName = 'This is a description of the energy level diagram...';
```

Or maybe we will create something new
```js
energyLevelDiagram.altText = 'This is a description of the energy level diagram...';
```

I don't see a scenery issue for deciding on the high level API for non-interactive content. I would like to create that in scenery and close this issue.  @kathy-phet is that OK with you?

**Comment by kathy-phet** [comment]
Making an issue in scenery for the high level API that can serve this type of function sounds good, thanks!

accessiblename doesn’t match the goal of providing a longer description (could be multiple sentences) of what is visible in the node. I like the api name you suggested. Discussion can continue over in the scenery issue.
________________________________

**Comment by kathy-phet** [comment]
Maybe you could just transfer this issue and re-title it.
